### PR TITLE
Tyriar was here

### PR DESCRIPTION
**Pull request by Tyriar**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -56,4 +56,6 @@ app.use(function(err, req, res, next) {
   res.render('error');
 });
 
+console.log('Tyriar was here');
```
Comment (Tyriar) [inline index.js]: Why would you think this is a good idea?

File: index.js [hunk 2]
```diff
@@ -56,4 +56,6 @@ app.use(function(err, req, res, next) {
   res.render('error');
```
Comment (Tyriar) [inline index.js]: This renders an error :i_wish_emojis_worked_here:

File: index.js [hunk 3]
```diff
@@ -56,4 +56,6 @@ app.use(function(err, req, res, next) {
...
 });
```
Comment (Tyriar) [inline index.js]: :wink:

File: index.js (hunk 1 above)
Comment (rebornix) [reply index.js]: +1

Comment (rebornix) [reply index.js]: +3

Comment (rebornix) [reply index.js]: +4

Comment (rebornix) [reply index.js]: +4

Comment (rebornix) [reply index.js]: +5

Comment (rebornix) [reply index.js]: +5

Comment (rebornix) [reply index.js]: my review

File: index.js (hunk 3 above)
Comment (rebornix) [reply index.js]: finish review

File: index.js (hunk 1 above)
Comment (rebornix) [reply index.js]: start review

Comment (queerviolet) [reply index.js]: r1

**Pull request closed without merging**